apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+
+
+
+## What is provisioned for XM Cloud 
+
+So, how does this work? Referencing the figure below, let’s walk through the different bits and pieces of XM Cloud.  
+![XM Cloud Architecture](https://sitecorecontenthub.stylelabs.cloud/api/public/content/21cbe922e28b4896b1692be7c85ae6c4?v=c1ed41cb) 
+
+ 
+To begin, XM Cloud contains a Content Management instance (CM / purple box) that publishes content to a geographically distributed delivery layer called *Edge* (yellow box). Edge offers a GraphQL endpoint to then query for the content coming from XM Cloud. Your app is typically connected with, and can run with whatever host you are rendering your front end on, such as Vercel, Netlify, AWS, Azure, or another vendor.  
+
+
+In order to utilize the WYSIWYG editing experience, an internal rendering host, or *editing host* (green box), is delivered with XM Cloud. This editing host is managed by Sitecore, and runs a Node server that can host any Node-based application.  
+
+The main way of accessing XM Cloud is through the *Sitecore Cloud Portal* (solid outlined box, 'Sitecore Cloud').  This GUI allows you to manage and access your different Sitecore Apps and Users. For an in-depth exploration of the Sitecore Cloud Portal, navigate [here](https://doc.sitecore.com/portal/en/developers/sitecore-cloud-portal/introduction-to-the-sitecore-cloud-portal.html).  
+
+When developing with XM Cloud, you can use the available build and deployment services (purple box). XM Cloud has these services built-in, so that you do not have to think about building or provisioning the XM Cloud instance, it is all done for you.  If you have more requirements than currently provided, you can set up your own build and deployment pipeline using the [XM Cloud CLI](https://doc.sitecore.com/xmc/en/developers/xm-cloud/walkthrough--creating-an-xm-cloud-project-using-the-sitecore-cli.html).  
+
+Lastly, the XM Cloud CLI allows you to manage your XM Cloud instance and deploy, instead of using the GUI. You can also serialize content items that represent developer artifacts from your XM cloud instance into your source code repository. To learn more about serialization with Sitecore, navigate [here](https://doc.sitecore.com/xmc/en/developers/xm-cloud/serialization-in-sitecore.html). 
+
+ Once the provisioning and  build process are finished, the deployment starts. The moment the deployment is finished, you can start using the new XM Cloud environment. Some actions run in the background to warm up the application..  
+
+**Congratulations!** You have successfully connected to XM Cloud and built your first project, that has an environment that is set up and ready to be used. The solution code is ready and connected to a CI/CD pipeline as well – these steps that can take days or weeks are completed within minutes here.  
```

Comment:
I like the 'ease of use' call out at the end, but I'm unclear as to what we are comparing against. Does it sometimes take weeks with XM Cloud? Are we comparing to platform? I think we need to be slightly more specific.

Reply:
No specific comparison other than to general CI/CD setup times. XM Cloud tooling is helping the user get started faster.

Reply:
Updated the text to attempt to be more clear about CI/CD and XMC helping.